assets/pages/showcase/components/breakingHeadlines.jsx

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import React from 'react';
+
+import ProductPageContentBlock from 'components/productPage/productPageContentBlock/productPageContentBlock';
+import BreakingTheHeadlines from 'components/svgs/breakingTheHeadlines';
+
+export default function BreakingHeadlines() {
+  return (
+    <ProductPageContentBlock type="grey">
+      <div className="component-product-page-text-block">
+        <BreakingTheHeadlines />
+        <p>
+          We pride ourselves on our breaking news stories, in-depth analysis and thoughtful opinion pieces.
+          But its not just the news desk that works round the clock. Across the world, our sports writers,
```

Comment:
should be 'But it's' and that's assuming we want to start a sentence with but

Reply:
done